rotkehlchen/api/v1/schemas.py

```diff
@@ -4399,3 +4399,13 @@ class DeletePremiumDeviceSchema(Schema):
 class EditPremiumDeviceSchema(Schema):
     device_identifier = NonEmptyStringField(required=True)
     device_name = NonEmptyStringField(required=True)
+
+
+class ConfigurationUpdateSchema(Schema):
+    loglevel = fields.String(
+        required=True,
+        validate=validate.OneOf(
+            choices=['TRACE', 'DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'],
```

Comment:
we have this list already in https://github.com/rotki/rotki/blob/cc49035bdae59c348fa3af63716230b3b29badec/rotkehlchen/args.py#L105. Would be a good idea to move to a variable and reuse it

Reply:
+ 1